### PR TITLE
Fix PHPUnit configuration

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  |no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -11,21 +11,18 @@
          beStrictAboutTestSize="true">
 
     <testsuites>
-        <testsuite name="ForgotPasswordBundle tests suite">
-            <directory>tests</directory>
+        <testsuite name="CoopTilleulsForgotPasswordBundle tests suite">
+            <directory>Tests</directory>
```
Comment (vincentchalamon) [inline phpunit.xml.dist]: `Tests` directory does not exist, but `tests` does

File: phpunit.xml.dist [hunk 2]
```diff
@@ -11,21 +11,18 @@
...
         </testsuite>
     </testsuites>
 
     <filter>
         <whitelist>
             <directory>.</directory>
             <exclude>
-                <directory>tests</directory>
+                <directory>Tests</directory>
```
Comment (vincentchalamon) [inline phpunit.xml.dist]: Same

File: phpunit.xml.dist [hunk 3]
```diff
@@ -11,21 +11,18 @@
...
                 <directory>features</directory>
                 <directory>vendor</directory>
-                <directory>src/Entity</directory>
```
Comment (vincentchalamon) [inline phpunit.xml.dist]: Directories `Entity` & `Resources` should be added in this whitelist

Comment (dunglas) [inline phpunit.xml.dist]: Why?

Comment (vincentchalamon) [inline phpunit.xml.dist]: Because there is no need to test those directories

Comment (dunglas) [inline phpunit.xml.dist]: Entities should (or at least can) be tested.
The `Resources` shouldn't contain PHP code anyway.

Comment (vincentchalamon) [inline phpunit.xml.dist]: In this project, `Entity` directory just presents 2 entities with only getters & setters. I don't see the need to test them. `Resources` directory only contains a `services.yml` file

Comment (dunglas) [inline phpunit.xml.dist]: Tests pass. I don't get the need to add them to the blacklist (except causing problems like this one).

**Pull request merged**